docs/usage/self-hosted-configuration.md

```diff
@@ -564,6 +564,14 @@ Example url: `redis://localhost`.
 
 ## repositories
 
+If you use a `config.js` file, then the `repositories` array can be an object:
```

Comment:
The one who was asking if this kind of config was possible was using a `config.js` file.
I don't know if you _must_ use a `config.js` file for the `repositories` array to be an object?

Maybe it will work with a differently named `.js` file as well? Or with some other config?

Please check if the new text is technically correct. 😉

Reply:
This works in any of the global config options (eg env, config.json), so `config.js` can be removed here.

Reply:
```suggestion
In all of the global config options the `repositories` array can be an object:
```

How about this then?